### ISSUE TITLE
Use ClusterIP services by default

### ISSUE THREAD

**Issue by ilyasotkov** [SnappyDataInc/spark-on-k8s#12]
First of all, thanks for sharing your code 👍

I noticed that the umbrella chart will create a bunch of Kubernetes LoadBalancer services by default, so each service will create a cloud load balancer with a unique IP address. To me it seems excessive to create a separate load balancer for every service, considering they are billable resources.

It would be better to use ClusterIP services by default (so you can access each service privately via `kubectl proxy`) That would also automatically provide us with the ability to expose the services publicly via Ingress resources (using nginx-ingress or any other ingress controller).

**Comment by jramnara** [comment]
good point about these being billable resources. We will look into this. 
Please feel free to submit a PR. Thanks.

**Comment by poshakjaiswal** [comment]
Any update on changing the chart
